pandas/_libs/tslibs/dtypes.pyx

```diff
@@ -304,8 +304,8 @@ cdef dict c_OFFSET_DEPR_FREQSTR = {
     "BQ-NOV": "BQE-NOV",
 }
 cdef dict c_OFFSET_TO_PERIOD_FREQSTR = OFFSET_TO_PERIOD_FREQSTR
-cdef dict c_REVERSE_OFFSET_DEPR_FREQSTR = {
-    v: k for k, v in c_OFFSET_DEPR_FREQSTR.items()
+cdef dict c_REVERSE_OFFSET_REMOVED_FREQSTR = {
+    v: k for k, v in c_OFFSET_REMOVED_FREQSTR.items()
```

Comment:
is it possible to remove this mapping completely? this would be a nice chance to clean up the code, is it still necessary to check whether an alias is one of the removed one? or is there some other deprecation that needs to happen first?

Reply:
yes, I think it's possible. I removed `c_REVERSE_OFFSET_REMOVED_FREQSTR` and changed the error message to "`{name} is not supported as period frequency"` to make it look alike as in other places where we get frequency invalid for period, but valid for offsets.